resources/developer-mode.mjs

```diff
@@ -12,12 +13,71 @@ export function createDeveloperModeContainer() {
     const content = document.createElement("div");
     content.className = "developer-mode-content";
     content.append(createUIForSuites());
+    content.append(document.createElement("hr"));
+    content.append(createUIForMeasurementMethod());
+    content.append(document.createElement("br"));
+    content.append(createUIForWarmupSuite());
+    content.append(document.createElement("br"));
+    content.append(createUIForIterationCount());
     details.append(content);
 
     container.append(details);
     return container;
 }
 
+export function createUIForMeasurementMethod() {
+    let check = document.createElement("input");
+    check.type = "checkbox";
+    check.id = "measurement-method";
+    check.checked = params.measurementMethod === "raf";
+
+    check.onchange = () => {
+        params.measurementMethod = check.checked ? "raf" : "timer";
+        fixupURL();
+    };
+
+    let label = document.createElement("label");
+    label.append(check, " ", "rAF timing");
+
+    return label;
+}
+
+export function createUIForWarmupSuite() {
+    let check = document.createElement("input");
+    check.type = "checkbox";
+    check.id = "warmup-suite";
+    check.checked = !!params.useWarmupSuite;
+
+    check.onchange = () => {
+        params.useWarmupSuite = check.checked;
+        fixupURL();
+    };
+
+    let label = document.createElement("label");
+    label.append(check, " ", "warmup suite");
+
+    return label;
+}
+
+export function createUIForIterationCount() {
+    let range = document.createElement("input");
+    range.type = "range";
+    range.id = "iteration-count";
+    range.min = 1;
+    range.max = 20;
+    range.value = params.iterationCount;
+
+    range.onchange = () => {
+        params.iterationCount = parseInt(range.value);
+        fixupURL();
+    };
+
+    let label = document.createElement("label");
+    label.append("iterations ", range);
```

Comment:
Todo - @flashdesignory mentioned adding the actual count alongside the slider

Reply:
Yeah, can we add the iteration count? Right now, I can't tell what number of iterations are picked.